Gemfile

```diff
@@ -23,4 +23,9 @@ else
   gem 'puppet', :require => false
 end
 
+# rspec must be v2 for ruby 1.8.7
+if RUBY_VERSION >= '1.8.7' and RUBY_VERSION < '1.9'
+  gem 'rspec', '~> 2.0'
```

Comment:
rspec 3.1 is the last rspec that works on ruby 1.8.7:

https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/Gemfile#L15